flytekit/loggers.py

```diff
@@ -33,8 +36,18 @@
 user_space_logger = child_loggers["user_space"]
 
 # create console handler
-ch = logging.StreamHandler()
```

Comment:
This is unfortunately a breaking change but I guess a minor version bump (1.6.0) is fine? We might want to highlight this in release notes.

Reply:
Cc @wild-endeavor / @eapolinario